gigasecond/src/main/kotlin/Gigasecond.kt

```diff
@@ -0,0 +1,15 @@
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.LocalTime
+
+class Gigasecond {
+    val date : LocalDateTime
+
+    constructor(date : LocalDate) {
+        var dateTime = LocalDateTime.of(date, LocalTime.of(0, 0, 0))
+        this.date = dateTime.plusSeconds(1000000000)
+    }
```

Comment:
```suggestion
    constructor(date : LocalDate): this(LocalDateTime.of(date, LocalTime.of(0, 0, 0)))
```
아래 더하기 로직을 재활용하는건 어떨까요?